tests/framework/installer/ceph_installer.go

```diff
@@ -887,8 +887,12 @@ spec:
                     - "-c"
                     - |
                       set -xEeuo pipefail
-                      # Wipe LVM data from disks
-                      #
+                      # Wipe a LVM that is created in testing OSD on PVC backed by LVM
```

Comment:
With #5452 This test wipe job is going away. Disk cleanup should now be part of the core rook cleanup instead of doing it only test code, unless it is test-only cleanup.
@leseb What is the status of getting that merged?

Reply:
I just need you to click approve :D
I'll rebase soon.

Reply:
I'll also take a look at #5452

Reply:
Since #5452 is merged, I'll update this PR.